SPEC.md

```diff
@@ -549,6 +551,16 @@ The AC Name Type is used as the primary key for a number of fields in the schema
 The schema validator will ensure that the keys conform to these constraints.
 
 
+### AC Version Type
+
+The App Container specification aims to follow semantic versioning and retain forward and backwards compatibility within major versions. For example, if an implementation is compliant against version 1.0.1 of the spec, it shall remain compatible with the complete 1.x series.
```

Comment:
Please split to single-sentence-per-line (we prefer this for reviewability right now)